### PR TITLE
Fixed being unable to upgrade grabs with hotkey.

### PR DESCRIPTION
**Pull request by naelstrof**
Deals with #7130

### CODE REVIEW

**Comment by kingofkosmos** [comment]
Do page down and ctrl+a  work too?

**Comment by naelstrof** [comment]
Ctrl+a doesn't seem to work for anything, page down upgrades the grip though.

**Comment by kingofkosmos** [comment]
Okay, cool. Guess I'll have to update the wiki hotkeys-page, that's where I got the ctrl+a.

File: code/modules/mob/mob_grab.dm [hunk 1]
```diff
@@ -115,6 +115,8 @@
 		affecting.Weaken(5)	//Should keep you down unless you get help.
 		affecting.losebreath = min(affecting.losebreath + 2, 3)
 
+/obj/item/weapon/grab/attack_self(mob/user)
+	s_click(hud)
```
Comment (Anonus) [inline code/modules/mob/mob_grab.dm]: This is awful. Why isn't all of s_click for this in attack self.

**Comment by PKPenguin321** [comment]
:+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1:

File: code/modules/mob/mob_grab.dm (hunk 1 above)
Comment (naelstrof) [inline code/modules/mob/mob_grab.dm]: Because without s_click you can't click on the grab to upgrade the grip. Both are required.

**Pull request merged**